x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/WaterfallWithSummmary/TraceSummary.tsx

```diff
@@ -0,0 +1,255 @@
+    color = theme.euiColorVis5;
+  } else if (status >= 400 || status < 500) {
+    color = theme.euiColorVis7;
+  } else if (status >= 500) {
+    color = theme.euiColorVis2;
+  }
+
+  return (
+    <EuiToolTip
+      content={i18n.translate('xpack.apm.transactionDetails.statusCode', {
+        defaultMessage: 'Status code'
+      })}
+    >
+      <EuiBadge color={color}>{status}</EuiBadge>
+    </EuiToolTip>
+  );
+};
+
+const HttpInfoBadge = styled(EuiBadge)`
+  max-width: 250px;
+`;
+
+interface HttpInfoProps {
+  transaction: Transaction;
+}
+const HttpInfo: React.FunctionComponent<HttpInfoProps> = ({ transaction }) => {
+  const method = (
+    idx(transaction, _ => _.http.request.method) || ''
+  ).toUpperCase();
+  const isRumAgent = isRumAgentName(transaction.agent.name);
+  const url = isRumAgent
+    ? idx(transaction, _ => _.transaction.page.url) || NOT_AVAILABLE_LABEL
+    : idx(transaction, _ => _.url.full) || NOT_AVAILABLE_LABEL;
+  const status = idx(transaction, _ => _.http.response.status_code);
+
+  return (
+    <>
+      <HttpInfoBadge title={undefined}>
+        <EuiToolTip
+          content={i18n.translate(
+            'xpack.apm.transactionDetails.requestMethodLabel',
+            {
+              defaultMessage: 'Request method'
+            }
+          )}
+        >
+          <MethodText>{method}</MethodText>
+        </EuiToolTip>{' '}
+        <EuiToolTip content={url}>
+          <>{url}</>
+        </EuiToolTip>
+      </HttpInfoBadge>{' '}
+      <HttpStatusBadge status={status} />
+    </>
+  );
+};
+
+interface ResultProps {
+  result: string;
+}
+const Result: React.FunctionComponent<ResultProps> = ({ result }) => {
+  return (
+    <EuiToolTip
+      content={i18n.translate('xpack.apm.transactionDetails.resultLabel', {
+        defaultMessage: 'Result'
+      })}
+    >
+      <EuiBadge color="default" title={undefined}>
+        {result}
+      </EuiBadge>
+    </EuiToolTip>
+  );
+};
+
+interface Props {
+  errorCount: number;
+  totalDuration?: number;
+  transaction: Transaction;
+}
+
+export const TraceSummary: React.FunctionComponent<Props> = ({
+  errorCount,
+  totalDuration,
+  transaction
+}) => {
+  const isHttp = 'http' in transaction;
+  const result = idx(transaction, _ => _.transaction.result);
+
+  return (
+    <>
+      <Heading />
+      <EuiText size="s">
+        <Timestamp transactionTimestamp={transaction['@timestamp']} />
+        <Separator />
+        <Duration
+          duration={transaction.transaction.duration.us}
+          totalDuration={totalDuration}
+        />
+        {(isHttp || result) && <Separator />}
+        {isHttp ? (
+          <HttpInfo transaction={transaction} />
+        ) : (
+          result && <Result result={result} />
+        )}
+        {errorCount > 0 && (
+          <>
+            <Separator />
+            <ErrorCountBadge
+              errorCount={errorCount}
+              includeText={true}
+              title={undefined}
```

Comment:
Why `title={undefined}`? Can't that be dropped?

Reply:
(Same with other occurrences of `title={undefined}`)

Reply:
An `EuiBadge` by default apparently sets the title attribute of the element to the contents of the element, so if you hover over "1 Error" you get a title tooltip that says "1 Error". This also occurs when an `EuiToolTip` is present, so you get a weird double-tooltip.